MaterialDesignThemes.Wpf/Themes/MaterialDesign3.ToggleButton.xaml

```diff
@@ -207,36 +207,31 @@
          TargetType="{x:Type ToggleButton}"
          BasedOn="{StaticResource MaterialDesignActionToggleButton}">
-    <Setter Property="Background" Value="{DynamicResource PrimaryHueLightBrush}" />
-    <Setter Property="BorderBrush" Value="{DynamicResource PrimaryHueLightBrush}" />
-    <Setter Property="Foreground" Value="{DynamicResource PrimaryHueLightForegroundBrush}" />
+    <Setter Property="Background" Value="{DynamicResource MaterialDesign.Brush.Primary.Light}" />
+    <Setter Property="BorderBrush" Value="{DynamicResource MaterialDesign.Brush.Primary.Light}" />
+    <Setter Property="Foreground" Value="{DynamicResource MaterialDesign.Brush.Primary.Light.Foreground}" />
   </Style>
 
   <Style x:Key="MaterialDesignActionDarkToggleButton"
          TargetType="{x:Type ToggleButton}"
          BasedOn="{StaticResource MaterialDesignActionToggleButton}">
-    <Setter Property="Background" Value="{DynamicResource PrimaryHueDarkBrush}" />
-    <Setter Property="BorderBrush" Value="{DynamicResource PrimaryHueDarkBrush}" />
-    <Setter Property="Foreground" Value="{DynamicResource PrimaryHueDarkForegroundBrush}" />
+    <Setter Property="Background" Value="{DynamicResource MaterialDesign.Brush.Primary.Dark}" />
+    <Setter Property="BorderBrush" Value="{DynamicResource MaterialDesign.Brush.Primary.Dark}" />
+    <Setter Property="Foreground" Value="{DynamicResource MaterialDesign.Brush.Primary.Dark.Foreground}" />
   </Style>
 
   <Style x:Key="MaterialDesignActionSecondaryToggleButton"
          TargetType="{x:Type ToggleButton}"
          BasedOn="{StaticResource MaterialDesignActionToggleButton}">
-    <Setter Property="Background" Value="{DynamicResource SecondaryHueMidBrush}" />
-    <Setter Property="BorderBrush" Value="{DynamicResource SecondaryHueMidBrush}" />
-    <Setter Property="Foreground" Value="{DynamicResource SecondaryHueMidForegroundBrush}" />
+    <Setter Property="Background" Value="{DynamicResource MaterialDesign.Brush.Secondary}" />
+    <Setter Property="BorderBrush" Value="{DynamicResource MaterialDesign.Brush.Secondary}" />
+    <Setter Property="Foreground" Value="{DynamicResource MaterialDesign.Brush.Secondary.Foreground}" />
   </Style>
 
-  <!-- Obsolete -->
-  <Style x:Key="MaterialDesignActionAccentToggleButton"
-         TargetType="{x:Type ToggleButton}"
-         BasedOn="{StaticResource MaterialDesignActionSecondaryToggleButton}"/>
-
   <Style x:Key="MaterialDesignFlatToggleButton" TargetType="{x:Type ToggleButton}">
-    <Setter Property="Background" Value="{DynamicResource MaterialDesignToolBackground}" />
+    <Setter Property="Background" Value="{DynamicResource MaterialDesign.Brush.ToolBar.Item.Background}" />
     <Setter Property="FontSize" Value="18" />
-    <Setter Property="Foreground" Value="{DynamicResource MaterialDesignToolForeground}" />
+    <Setter Property="Foreground" Value="{DynamicResource MaterialDesign.Brush.ToolBar.Item.Foreground}" />
```

Comment:
Target type here is `ToggleButton` so should the resource key be something like this?

Or perhaps one of the more generic/common keys instead?

```suggestion
    <Setter Property="Foreground" Value="{DynamicResource MaterialDesign.Brush.ToggleButton.Foreground}" />
```